helm/korifi/values.yaml

```diff
@@ -44,6 +44,16 @@ api:
       idle: 900
       readHeader: 10
 
+  infoConfig:
+    build: ""
+    description: ""
+    name: ""
+    version: 0
+    minCLIVersion: ""
```

Comment:
I wonder whether it makes sense to default cli versions to something? No idea what default values should be though.

Reply:
On a second glance, it seems that we already have some ["min version" code](https://github.com/cloudfoundry/korifi/blob/c353c9ecd6f55a14799d22bedfaf5a822d15d8cc/api/middleware/cli_version.go#L14) in place. I guess returning this constant in the `/v3/info` presenter would make sense, moreover it is already used to [error](https://github.com/cloudfoundry/korifi/blob/c353c9ecd6f55a14799d22bedfaf5a822d15d8cc/api/middleware/cli_version.go#L41) if cli version is tool low.

Reply:
For cf-deployment, those properties seem to be empty.

Some real-life output:
```
{"build":"v1.56.0","cli_version":{"minimum":"","recommended":""},"custom":{},"description":"","name":"cf-deployment","version":39,"links":{"self":{"href":"https://api.system.EXAMPLE.com/v3/info"},"support":{"href":"support@EXAMPLE.com"}}}
```

I think I would keep minimum and recommended version empty by default and let the operator decide how to use it.

For example if you would fix `cf whoami` ( https://github.com/cloudfoundry/korifi/issues/1724 ), you would probably require a new CF CLI version and the operator could point out the fact you need a specific CLI version.

I consider 
https://github.com/cloudfoundry/korifi/blame/c353c9ecd6f55a14799d22bedfaf5a822d15d8cc/api/middleware/cli_version.go#L14
to be a hard requirement the Korifi release imposes, but the operator might be more strict.